.github/workflows/release-testing.yaml

```diff
@@ -0,0 +1,125 @@
+name: release testing artifacts
+
+on:
+  push:
+    branches:
+      - testing
```

Comment:
this is the branch with patches applied right? this workflow isn't picking patches?